### PR TITLE
DSD-611: Button NoBrand

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-611](https://jira.nypl.org/browse/DSD-611)

## This PR does the following:
- Add the `NoBrand` variant for the `Button` component.

## How has this been tested?

Storybook

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] View [the example in Storybook]()

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr770-p5pvfy8vwqah7b49zztpo5umuggtv0iw.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61966d96de4970639ae404a2

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr770-p5pvfy8vwqah7b49zztpo5umuggtv0iw.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61966d96de4970639ae404a2

File: src/theme/components/button.ts [hunk 1]
```diff
@@ -100,11 +100,19 @@ const searchBar = {
   borderRightRadius: { base: "none", md: "sm" },
   lineHeight: "1.75",
 };
+const noBrand = {
+  ...primary,
+  bg: "ui.black",
+  color: "ui.white",
+  _hover: {
+    bg: "ui.black",
+    opacity: "0.8",
```
Comment (bigfishdesign13) [inline src/theme/components/button.ts]: For the `hover` state, remove `opacity` and change `bg` to `ui.gray.xdark`.

**Review by bigfishdesign13 — CHANGES_REQUESTED** [review]
Just one small change for the hover state.

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr770-p5pvfy8vwqah7b49zztpo5umuggtv0iw.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61966d96de4970639ae404a2

**Review by bigfishdesign13 — APPROVED** [review]
All good.

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr770-p5pvfy8vwqah7b49zztpo5umuggtv0iw.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61966d96de4970639ae404a2

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr770-p5pvfy8vwqah7b49zztpo5umuggtv0iw.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61966d96de4970639ae404a2

**Pull request merged**